main.go

```diff
@@ -14,6 +15,7 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/mitchellh/hashstructure"
```

Comment:
I would like to keep dependencies to a minimum. Is there any other way we can hash? Maybe implementing Stringer on container or something?